node/core/dispute-coordinator/src/db/v1.rs

```diff
@@ -47,6 +65,44 @@ impl DbBackend {
 	pub fn new(db: Arc<dyn Database>, config: ColumnConfiguration) -> Self {
 		Self { inner: db, config }
 	}
+
+	/// Cleanup old votes.
+	///
+	/// Should be called whenever a new ealiest session gets written.
```

Comment:
```suggestion
	/// Should be called whenever a new earliest session gets written.
```

Reply:
Hmm, I thought I fixed that already. Anyway, thanks!